### PR TITLE
Dynamic secret configuration added to fluentd configmap

### PR DESCRIPTION
**Pull request by vashisthk1**
-vars added in values file 
-secretref key
-dynamic password
-dynamic ingress points

### CODE REVIEW

File: stable/fdi-eck/templates/deploy/elastic.yaml [hunk 1]
```diff
@@ -47,3 +55,6 @@ spec:
         resources:
           requests:
             storage: 128Gi
+
+
+
```
Comment (wg102) [inline stable/fdi-eck/templates/deploy/elastic.yaml:60]: Not strictly necessary but please clean up the files to format. One empty line is usual standard

File: stable/fdi-eck/values_minikube.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  elasticSearchUrl: "elasticSearchUrl"
+  elasticSearchUsername: "elastic"
+  elasticSearchPassword: ""
+  filter: |
+    <filter **>
+        @type grep<regexp>  key $.kubernetes.container_name  pattern  (^.*dotstatsuite.*$|^.*istio-proxy.*$)</regexp>
+        <exclude>  key log  pattern (^.*xdsproxy.*$|^.*Microsoft.*$|^.*cached.*$|^.*/health.*$|^.*/live.*$|^.*Request finished in.*$)</exclude>
+    </filter>
+elastic:
+  password: "elasticPassword"
```
Comment (wg102) [inline stable/fdi-eck/values_minikube.yaml:14]: Hardcoded password?

File: stable/fdi-eck/values.yaml [hunk 3]
```diff
@@ -1,19 +1,20 @@
 ci_slug: "fdi-meta"
-namespace: "fdi"
+namespace: "kube-system"
 configMap:
   name: "daaas-fdi-fluentd-config" #fdi namespace use daaas-fdi-fluentd-config aaw use org-fdi-system-fluentd-config
   elasticSearchUrl: "elasticSearchUrl"
   elasticSearchUsername: "elastic"
-  elasticSearchPassword: ""
+  elasticSearchPassword: "k8s-cancentral-01-development"
```
Comment (rohank07) [inline stable/fdi-eck/values.yaml:7]: Open-text password - use K8s secret

**Review by wg102 — CHANGES_REQUESTED** [review]
Hello, I did a quick review. If you could clarify the concerns about the password.

File: stable/fdi-eck/values.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (vashisthk1) [reply stable/fdi-eck/values.yaml]: Removed the passwords, was not working.

File: stable/fdi-eck/values_minikube.yaml (hunk 2 above)
Comment (vashisthk1) [reply stable/fdi-eck/values_minikube.yaml]: removed, unable to use

**Review by wg102 — APPROVED** [review]

**Comment by vashisthk1** [comment]
Hi, please merge the request now. Fixed the bug. 

Thank you for your help!

**Review by Jose-Matsuda — APPROVED** [review]

**Comment by Jose-Matsuda** [comment]
You can Squash and Merge at your convenience
image

**Comment by vashisthk1** [comment]
> You can Squash and Merge at your convenience image

Hi, 

I cannot see this option. Is it a permissions issue?

If possible can you please squash and merge?

Thanks,
Kartik

**Pull request merged**